index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Ingrese su nombre');
+alert(`Hola ${nombreUsuario}!`);
```

Comment:
En este caso me parece que no serviría la interpolación
  ```suggestion
alert('Hola' + nombreUsuario);
```